### DISCUSSION TITLE
Record stops please help

### DISCUSSION THREAD

**Discussion by mohotta** [SeaDve/Kooha#238] [Q&A]
Failed to start recording

Caused by:
    0: Check out <a href="https://github.com/SeaDve/Kooha#-it-doesnt-work">It Doesn't Work page</a> for help.
    1: Failed to create ScreencastSession
    2: Failed to create session
    3: Failed to call `CreateSession` with parameters: Variant { ptr: 0x55632c37b1c0, type: VariantTy { inner: "(a{sv})" }, value: "({'handle_token': <'kooha_3'>, 'session_handle_token': <'kooha_2'>},)" }
    4: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.ScreenCast” on object at path /org/freedesktop/portal/desktop

**Comment by jgarte** [comment]
Any updates on this? @mohotta
